libc/startup/baremetal/aie2/crt1.cc

```diff
@@ -6,10 +6,10 @@
 // (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
 
 extern "C" {
-extern int _main(int, char **);
+extern int main(int, char **);
 void _Exit(int val) {
   (void)val;
   done();
 }
-void _main_init() { _Exit(_main(0, nullptr)); }
```

Comment:
Is this compatible with chess?  My recollection here is having to do some awkward things to get the same code to compile with chess and peano.

Reply:
The main problem I see would be the organization of static constructors, which are currently left out. In this set-up we control both startup code and _Exit, and compatibility hinges on C 'main' being compiled to link 'main'.
What I have seen is that the structure of static constructors is compatible, i.e. an array of pointers to void(void) initialization routines. When we want to add them I hope to glue them together by a symbol equate in the ldfile.
I don't think we want to add argc/argv at this point.
If there are any insights I've missed, please let me know.